pkg/cloudclient/aws/private_test.go

```diff
@@ -134,6 +137,8 @@ func TestRemoveAWSELB(t *testing.T) {
 			if err != nil {
 				t.Fatalf("Couldn't reload the test machine (named %s): %v", machineInfo.Name, err)
 			}
+			newMachines = append(newMachines, machine)
```

Comment:
Curious why we need to do this when we didn't before?

Reply:
https://github.com/kubernetes-sigs/controller-runtime/blame/15b6689c55745514d6bade4c55efb3333c0da4d9/pkg/client/fake/client.go#L358 There is a change in FakeKubeClient(after migrate) which will clear the value in the old machine pointer. 
So it is not needed in the existing code, but will get blank machineList with the new controller-runtime's fakeClient after `Get()`